linux_os/guide/system/software/integrity/fips/sysctl_crypto_fips_enabled/rule.yml

```diff
@@ -19,8 +19,6 @@ rationale: |-
 
 severity: high
 
-platform: machine  # The oscap sysctl probe doesn't support offline mode
```

Comment:
Limitation of OpenSCAP. Content needs to be tool agnostic. Reverting.

Reply:
ditto (see `sysctl_kernel_exec_shield`)